source/channels/make-calls.rst

```diff
@@ -42,11 +42,18 @@ The call link is valid for long as the channel is active. When a channel is arch
 Record calls
 ------------
 
-The host of a meeting can record the call. The recording includes audio as well as any screen sharing during the call. The default setting for a recording is 60 minutes, but your system admin may adjust that as needed. Ten minutes before the time limit, the host will receive a reminder that the recording limit will be reached soon.
+From Mattermost v7.7 if you're the host of a meeting, you can record the call. 
+
+The recording includes audio as well as any screen sharing during the call. The default setting for a recording is 60 minutes, but your system admin may adjust that as needed. Ten minutes before the time limit is reached, you'll receive a reminder that the recording limit will be reached soon.
 
 If your call is going to continue and you still want to record, once the first recording is complete you can start a new one immediately after.
 
-Call recordings are posted by the Calls bot directly in the call thread as normal file attachment and can be accessed by all users in the channel.
+When you stop recording, the recording is posted in the call thread as a file attachment. It's available to all users in the channel both during the call, and when the call has ended.
+
+To record a call:
+
+1. Select **Start call** in the header of the channel, group message, or direct message.
+2. When the call has started, use the slash command
 
 Frequently asked questions
```

Comment:
@streamer45, these were added in the first draft of the docs but I think they'd work better in the deployment section - what do you think? Otherwise this page is very mixed between end-user and admin. We can also rename the deployment section to "Calls deployment and configuration" to be more suitable for this additional content.